src/app/domains/wallets/domains.wallets.gateway.ts

```diff
@@ -14,18 +16,24 @@ export class DomainsWalletsGateway {
   }
 
   public getWallets(): Observable<IWalletApiResponse[]> {
-    return this.http.get<IWalletApiResponse[]>(API_WALLETS_URL);
+    return fakeRequest(GET_WALLETS_API_RESPONSE_MOCK());
+    //return this.http.get<IWalletApiResponse[]>(API_WALLETS_URL); // TE ZMIANY SA CHWILOWE, BO MI DALEJ BACKEND NIE DZIALA
```

Comment:
Can be removed for now, anyway backend has not yet implemented transaction endpoints so we need to go for the `fakeRequest` approach

Reply:
Not done yet